crates/iceberg-catalog-bin/src/serve.rs

```diff
@@ -160,3 +171,23 @@ async fn build_nats_client(nat_addr: &Url) -> Result<NatsBackend, Error> {
     };
     Ok(nats_publisher)
 }
+
+fn build_kafka_producer(kafka_brokers: &Vec<SocketAddr>) -> Result<KafkaBackend, Error> {
+    let kafka_brokers_csv = kafka_brokers
+        .iter()
+        .fold(String::new(), |acc, e| format!("{acc},{e}"));
```

Comment:
Doesn't this prepend a comma to the string?

Initially, `acc` is empty, so the first iteration would yield: `,<some-address>`

I think we already depend on itertools, so you could just use https://docs.rs/itertools/latest/itertools/fn.join.html instead